apps/vessel-history/pages/_document.tsx

```diff
@@ -201,12 +190,13 @@ class MyDocument extends Document {
           </head>
         </Head>
         <body>
+          <div id="root"></div>
```

Comment:
I think we can avoid having it here by these two options:
- [use the appSelector prop](https://github.com/GlobalFishingWatch/frontend/blob/develop/apps/fishing-map/features/app/App.tsx#L229) in the Modal ui-component
- update [ui-component default value](https://github.com/GlobalFishingWatch/frontend/blob/develop/libs/ui-components/src/modal/Modal.tsx#L29) as CRA is not used anymore (or keep it as it is and add another fallback value for next apps)

what do you prefer?

Reply:
Good catch, thanks! I'll use the `appSelector="__next"` in the Modal ui-component